cvat/apps/dataset_manager/_tests.py

```diff
@@ -0,0 +1,309 @@
+
+# Copyright (C) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+class _GitImportFix:
+    import sys
+    former_path = sys.path[:]
+
+    @classmethod
+    def apply(cls):
+        # HACK: fix application and module name clash
+        # 'git' app is found earlier than a library in the path.
```

Comment:
@zhiltsov-max ,

Can we just rename "git" application? Let's call it "dataset_repo". In the future it will be cool to have the repo in datumaro format.

Reply:
@azhavoro, @nmanovic,

I don't know if it is really easier. I suppose just renaming the application and imports is not enough regarding to django. If I rename an app, should I also rename everything in migrations and special files like `app/apps.py`?

Reply:
@zhiltsov-max , it isn't easy. I believe we need a separate merge request to re-name git app. @azhavoro , could you please prepare such PR?

Reply:
I don't think it is really needed now, as the issue only affects tests. Renaming is described [here](https://stackoverflow.com/questions/8408046/how-to-change-the-name-of-a-django-app), and it looks like it requires non-standard migration steps, which can complicate version upgrade process for little benefits.

Reply:
@zhiltsov-max , it is definitely don't need for your PR. But in general we need to fix the issue. Workarounds like _tests.py are not acceptable.

Reply:
https://github.com/opencv/cvat/issues/1329

Reply:
Agree, need to rename git app.